Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# On your host:
+# git clone https://github.com/fedora-infra/bodhi.git
+# cd bodhi
+# copy this file here
+# vagrant up
+# vagrant ssh -c"cd /vagrant/; pserve development.ini --reload"
+
+Vagrant.configure(2) do |config|
+  config.vm.box_url = "http://cloud.centos.org/centos/7/vagrant/x86_64/images/CentOS-7.LibVirt.box"
+  config.vm.box = "centos7-minimal-libvirt"
+  config.vm.network "forwarded_port", guest: 8080, host: 8080
+  config.vm.network "forwarded_port", guest: 15672, host: 15672
+  config.vm.synced_folder ".", "/vagrant", type: "sshfs"
+
+  config.vm.provider :libvirt do |domain|
+      domain.cpus = 2
+      domain.graphics_type = "spice"
+      # The unit tests use a lot of RAM.
+      domain.memory = 2048
```

Comment:
2048 is fine, I just wanted to make sure it wasn't a leftover from the Bodhi Vagrantfile. Since it is the program and not the unit tests, can you either adjust or remove the comment about the unit tests?